tests/cpp_validation_tests/CMakeLists.txt

```diff
@@ -11,7 +11,8 @@ add_executable(power_grid_model_validation_tests ${PROJECT_SOURCES})
 
 target_link_libraries(power_grid_model_validation_tests
     PRIVATE
-        power_grid_model
+        power_grid_model_c
```

Comment:
you can optionally delete this cause `power_grid_model_cpp` already exposes `power_grid_model_c` in the interface
```suggestion
```

Reply:
I removed it. No point in keeping the C API if we have the Cpp API.